protobuf-provider/src/main/java/io/confluent/kafka/schemaregistry/protobuf/ProtobufSchema.java

```diff
@@ -157,7 +166,7 @@ public class ProtobufSchema implements ParsedSchema {
   protected static final String CONFLUENT_PREFIX = "confluent.";
   private static final String CONFLUENT_FILE_META = "confluent.file_meta";
   private static final String CONFLUENT_MESSAGE_META = "confluent.message_meta";
-  private static final String CONFLUENT_FIELD_META = "confluent.field_meta";
+  protected static final String CONFLUENT_FIELD_META = "confluent.field_meta";
```

Comment:
Let's change the other CONFLUENT_X_META fields from private to protected for consistency.